product.json

```diff
@@ -4,6 +4,7 @@
 	"applicationName": "code-oss",
 	"dataFolderName": ".vscode-oss",
 	"win32MutexName": "vscodeoss",
+	"enableTelemetry": true,
```

Comment:
maybe debatable but it helps with local development

Reply:
I would leave this out, and enable it on demand.